tests/system/test_client.py

```diff
@@ -1485,33 +1484,6 @@ def test_copy_table(self):
         got_rows = self._fetch_single_page(dest_table)
         self.assertTrue(len(got_rows) > 0)
 
-    def test_get_set_iam_policy(self):
```

Comment:
This test is now redundant with a code sample which serves the same purpose with regards to testing.